dev-env/ssh-config

```diff
@@ -0,0 +1,11 @@
+Host default
+  HostName 127.0.0.1
+  User vagrant
+  Port 2222
+  UserKnownHostsFile /dev/null
+  StrictHostKeyChecking no
+  PasswordAuthentication no
+  IdentityFile C:/Users/natha/work/Kive/dev-env/.vagrant/machines/default/virtualbox/private_key
```

Comment:
Did you mean to commit this file? This line particularly looks like it's specific to your workstation.

Reply:
🤦‍♂️ Good catch, thanks.